web/pgadmin/browser/server_groups/servers/pg_timetable/tasks/tests/test_pgt_task_put.py

```
@@ -0,0 +1,63 @@
+        """This function will update pgTimetable chain task"""
+
+        self.data['task_id'] = str(self.task_id)
+
+        if self.is_positive_test:
+            response = tasks_utils.api_put(self)
+
+            utils.assert_status_code(self, response)
+        else:
+            if self.mocking_required:
+                with patch(self.mock_data["function_name"],
+                           side_effect=[eval(self.mock_data["return_value"])]):
```

Ruff (0.15.21)

[error] Use of possibly insecure function; consider using `ast.literal_eval`

(S307)